src/assets/sass/mixins/_utilits.mixins.scss

```diff
@@ -1,12 +1,13 @@
-@use '../variables/variables.fonts' as ft;
```

Comment:
Can we keep singe quote here with prettier?
It's more convinient to type single quote rathet then double one